docs/getting-started/custom-ui-ory-elements.mdx

```diff
@@ -69,19 +80,48 @@ Follow these steps to get the sample application code and run it on your machine
    ````mdx-code-block
 
    <Tabs groupId="react-or-preact">
-     <TabItem value="unix" label="Unix systems" default>
+     <TabItem value="react" label="React" default>
 
-   ```shell-session
-   export VITE_ORY_SDK_URL=http://localhost:4000
-   ```
+    ```bash
+    # Unix
+    export VITE_ORY_SDK_URL=http://localhost:4000
+    ```
 
-     </TabItem>
-     <TabItem value="powershell" label="Windows Powershell">
+    ```powershell
+    # Windows
+    $Env:VITE_ORY_SDK_URL = "https://{your-project-slug-here}.projects.oryapis.com"
+    ```
 
-   ```powershell
-   $Env:VITE_ORY_SDK_URL = "https://{your-project-slug-here}.projects.oryapis.com"
-   ```
+    </TabItem>
+     <TabItem value="preact" label="Preact">
 
+    ```bash
+    # Unix
+    export VITE_ORY_SDK_URL=http://localhost:4000
+    ```
+
+    ```powershell
+    # Windows
+    $Env:VITE_ORY_SDK_URL = "https://{your-project-slug-here}.projects.oryapis.com"
+    ```
+
+    </TabItem>
+     <TabItem value="nextjs" label="Next.js">
+
+    ```bash
+    # Unix
+    export NEXT_PUBLIC_ORY_SDK_URL=http://localhost:4000
+    ```
+
+    ```powershell
+    # Windows
+    $Env:NEXT_PUBLIC_ORY_SDK_URL = "http://localhost:4000"
+    ```
+    ```bash
+    # Here we are using the Ory CLI to connect to Ory Network APIs.
+    ory tunnel http://localhost:3000 --project <project-slug> --dev
+    ```
+    [Learn more about environment variables with Next.js](https://nextjs.org/docs/basic-features/environment-variables)
```

Comment:
Please revert this section to the way it's on master right now. There's no need to have 3 tabs with the same two commands. That comment about using Ory Tunnel you added in the Next.js tab is there anyway - look at step 4. The note about environment variables with Next.js is delivered in the wrong way and is detached from the content. I don't see what calls for such a link. Please remove it. 

<img width="744" alt="image" src="https://user-images.githubusercontent.com/29116492/213114982-8573a12a-4621-49a5-8451-06fa6bc24c9e.png">

Reply:
Hi Tom,

I've removed the Next.js link as requested and also reverted the section to how it was before.

I'd just like to note a couple things:

- Environment variables in Next.js do not work the same way as they do in React/Preact. In order for the environment variable to be seen, we have to do it like [this](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser). Since the export is different, I figured having a showcase would help reduce questions from users. This is also why I included a link to the documentation, as it explains the difference.
- Also, for the redundant info, thank you for spotting it. Trying to fit two code Tab sections inside of each other didn't make sense, so that was why I changed it how I did, but in the process I missed Step 4.

The current commit should be good now, but we may get a lot of questions about the export issue.

Let me know if anything else is needed,
Thanks!

Reply:
Thanks for changing that! After reading your comment I noticed that the variable name for next.js is different than for the other 2 frameworks. 

I don't understand much from that link about env vars :D but I was thinking - isn't this just as simple as having all three sample apps use the same env var name?

Reply:
Hi @tomekpapiernik 

you cannot. the nextjs framework always prefixes `NEXT_` to the env variable.

Reply:
Also, I just noticed something off: the Unix variable points to localhost, Windows one is supposed to use actual project URL. Is that correct? @Benehiko @GabeCurran ? If that's wrong then please fix it in this PR, too. Thanks!

Reply:
It should be local url. 
We cannot use the project URL in SPAs since they will have CORS problems. rather show localhost and we explain the topic of using your domain in a "going to production" document.

Reply:
Ok, then @GabeCurran please also make sure that in both cases localhost is exported to an env var :)